mslib/msui/qt5/ui_mscolab_window.py

```diff
@@ -297,3 +293,13 @@ def retranslateUi(self, MSSMscolabWindow):
         self.actionCloseWindow.setText(_translate("MSSMscolabWindow", "CloseWindow"))
         self.actionCloseWindow.setShortcut(_translate("MSSMscolabWindow", "Ctrl+W"))
 
+
```

Comment:
None of the others  ui py files has this. Was this made by an additional option? Do you mind to create it without these lines?

Reply:
Just realized, I didn't add this. Let me check.

Reply:
@ReimarBauer please check if it looks fine now. Looks like I was using an unnecessary parameter earlier while generating it.